pom.xml

```diff
@@ -72,10 +72,9 @@
 		<dependency>
 			<groupId>life.qbic</groupId>
 			<artifactId>omero-client-lib</artifactId>
-			<version>1.2.0</version>
+			<version>1.2.1-SNAPSHOT</version>
```

Comment:
you need to release this dependency first, no snapshots are allowed

Reply:
ok, then I need to get 1.2.1 released

Reply:
done